backend-rust/src/graphql_api/baker.rs

```diff
@@ -127,17 +129,52 @@ impl Baker {
             .transpose()?
             .map(|c| AmountFraction::new_unchecked(c).into());
 
+        let total_stake: i64 =
+            sqlx::query_scalar!("SELECT total_staked FROM blocks ORDER BY height DESC LIMIT 1")
+                .fetch_optional(pool)
```

Comment:
We expect it to be there, so maybe:
```suggestion
                .fetch_one(pool)
```